src/renderer/components/dex/OrderForm.vue

```diff
@@ -232,10 +232,13 @@ export default {
       }
     },
     estimate() {
+      const holding = this.$store.state.currentMarket ?
+        this.$services.neo.getHolding(this.$store.state.currentMarket.baseAssetId).unitValue :
+        0;
+
       try {
         return new BigNumber(this.total).multipliedBy(
-          new BigNumber(this.$store.state.currentMarket ?
-            this.$services.neo.getHolding(this.$store.state.currentMarket.baseAssetId).unitValue : 0));
+          new BigNumber(parseFloat(holding)));
```

Comment:
It seems like we might want to do parseFloat earlier on, like in neo.js where it is set perhaps?
`holdingBalance.unitValue = val[`price_${lowercaseCurrency}`];`